design-patterns/domain-driven-design/pattern2/README.md

```diff
@@ -0,0 +1,205 @@
+# Pattern 2: Single Domain, Multiple Bounded Contexts, and a Single Uni
+
+**Prerequisite:** Please familiarize yourself with the Pattern 1 before you practice the Pattern 2.
+
+
+## Scenario
+
+In the Pattern 1 CRM Team created a Uni for the `CustomerAccount` entity. Let’s say requirements were different. We will remodel Uni with the Pattern 2. The CRM Team now wants to create another entity to track customer profiles(likes, habits etc.). The Marketing Team is interested in the customer profile dataset but doesn't want to build a separate Uni. Marketing will use customer profile data for customer research but expect the CRM Team to continue to manage and own the customer profile dataset.
+
+## Modeling
+
+In this scenario you will notice two separate domains. CRM domain and the Marketing domain. Since, Marketing Team doesn’t want to own any data and expects the CRM domain to build and own the `CustomerProfile` along with the `CustomerAccount`, you will model all datasets in the same Uni. You will notice two Bounded Contexts:`CustomerAccount` and `CustomerProfile`. However, Marketing Team will only do Read-Only activities on the `CustomerAccount` and `CustomerProfile` Bounded Contexts in the Uni so the CRM Team decided to provide a Read-Only copy to the Marketing Team for customer research and analytics. 
+
+If you are confused between `Bounded Context`, `Entity`, and the `Aggregate`. These terms correlate 1:1 for the purpose of this exercise. I will use the term `Entity` here to mean `Root Entity` which is an `Aggregate` in DDD. And, `Bounded Context` can have one or more than one `Entity`. 
```

Comment:
Maybe a link to a DDD resource or your blog?

Reply:
I will make a note and come back and edit it after I have a commit on main branch